packages/serverless/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
We are using the MIT License for new packages. See https://github.com/getsentry/sentry-javascript/blob/master/packages/angular/LICENSE

Reply:
Changed to 'MIT'

Reply:
It's still BSD here.